README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01 
+
+## Dimitrios Tanoglidis - Brief Bio
+
+![My photo](https://github.com/dtanoglidis/hw01/blob/master/Prof_phot.jpg)
```

Comment:
Nice inclusion of the photo in the repo to publish it. You can also use simply `![My photo](Prof_phot.jpg)` to embed it, since the README is in the same folder as the image